package-lock.json

```diff
@@ -64,7 +64,6 @@
       "resolved": "https://registry.npmjs.org/@babel/helper-annotate-as-pure/-/helper-annotate-as-pure-7.0.0.tgz",
       "integrity": "sha512-3UYcJUj9kvSLbLbUIfQTqzcy5VX7GRZ/CCDrnOaZorFFM01aXp1+GJwuFGV4NDDoAS+mOUyHcO6UD/RfqOks3Q==",
       "dev": true,
-      "optional": true,
```

Comment:
This is sometimes a sign of an older version of npm which could lead to lock file ping-pong if we're not careful. Check whether there's a newer version available and if so update the lock file after updating npm.

Reply:
I moved to 6.4.1 sometime around the holidays when there was talk of unifying NPM versions across development machines.  Looks like 6.10.1 is out.  This might, however, be something from Eli's machine (since I based my code on his initial `sensor-block` branch which also included the Rete packages).

Reply:
I'm about to merge a bunch of dependency changes to master with 6.10, so you might update to 6.10 and then run npm install to see if there are changes.

Reply:
Definitely looks to be something that my machine did.

Reply:
Reverted to the original changes in the `sensor-block` branch.  Need to investigate why my machine is toggling these to true.

Reply:
Actually, your machine is removing the "optional: true", which is probably just due to npm version.

Reply:
Right.